mr_adc_compute.py

```diff
@@ -62,6 +59,13 @@ def kernel(mr_adc):
     # Compute amplitudes
     mr_adc_amplitudes.compute_amplitudes(mr_adc)
 
+    # Compute CVS integrals and amplitudes
+    if mr_adc.method_type == "cvs-ip":
+        mr_adc_integrals.compute_cvs_integrals_2e_incore(mr_adc)
```

Comment:
I think integrals should be computed in mr_adc.py, not in the compute function

Reply:
This function slices the CVS integrals, replacing general core indices for CVS core and valence. I can move this to the mr_adc.py file, but it will require a function to delete core indices variables. It can be added in the same place where compute_cvs_integrals_2e_incore() is now. What do you think?

Reply:
Yes, we should move compute_cvs_integrals_2e_incore to mr_adc.py, then add a function to delete integrals that are no longer necessary at the end of amplitude function where 4-RDM is deleted.